docs/skills/infra-md-to-nextjs-migration/SKILL.md

```
@@ -263,8 +267,8 @@ Markdown内の「参考やリンク」セクション（公式ドキュメント
 
 ## 進捗の確認
 
-セッションをまたぐ場合は、必ずルートディレクトリの `MIGRATION_PROGRESS.md` を確認・更新し、進捗とコンテキストを引き継ぐこと。
+セッションをまたぐ場合は、必ずプロジェクトルートの `MIGRATION_PROGRESS.md` を確認・更新し、進捗とコンテキストを引き継ぐこと。
 
 既存の進行状況管理:
 
-- MIGRATION_PROGRESS.md
+- MIGRATION_PROGRESS.md
```

markdownlint-cli2 (0.22.1)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)